WebHostLib/templates/playerTracker.html

```diff
@@ -0,0 +1,176 @@
+{% block head %}
```

Comment:
This file is kind of hard to read. Not your fault necessarily, but simplification of it may be a good goal in the future (of the project or of this PR).

Reply:
Not too much I can do as far as simplifying the appearance due to the way I'm rendering it, but i did add some spacing and comments to make it a bit easier to read through.